build

```diff
@@ -1,7 +1,5 @@
 #!/bin/sh
 
-set -ef
```

Comment:
If the version/name scripts don't return non-zero exit codes anymore, then we can keep this I think.